src/js/base/post-load-hooks.js

```diff
@@ -75,10 +75,21 @@ define("pyret-base/js/post-load-hooks", function() {
           empty_only : runtime.getField(ds, "empty-only")
         };
 
-        runtime["makeCStr"] = runtime.getField(ds, "c-str").app;
-        runtime["makeCNum"] = runtime.getField(ds, "c-num").app;
-        runtime["makeCBool"] = runtime.getField(ds, "c-bool").app;
-        runtime["makeCCustom"] = runtime.getField(ds, "c-custom").app;
+        function reCache(src, constructorName, target, cachedName) {
```

Comment:
This seems unrelated to the point of the PR, unless I'm missing something. But this code is already on horizon, so maybe this is irrelevant.